saml2/saml2-service-provider/src/main/java/org/springframework/security/saml2/provider/service/authentication/Saml2AuthenticationRequestContext.java

```diff
@@ -87,6 +113,41 @@ public String getRelayState() {
 		return this.relayState;
 	}
 
+	/**
+	 * Returns the desired ({@code AuthnContextClassRef} that this SP wishes to set in the
```

Comment:
While SP and "relying party" are both acceptable terms. For consistency with `RelyingPartyRegistration`, would you please use "relying party" instead?